src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -1057,4 +1063,7 @@ def _make_singular(value):
         g.command('delete', 'delete')
         g.show_command('show', 'get')
         g.command('list', 'list')
+
+    with self.command_group('network service-aliases', network_service_aliases_sdk) as g:
+        g.custom_command('list', 'list_service_aliases')
```

Comment:
Should command group with only one command convert to list-xxxx ?

Reply:
yes. according to our doc it should. I have asked service team about this command whether there will be more commands in the future or not. So do not merge now.